applications/client/src/main/java/com/paritytrading/philadelphia/client/command/WaitCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.paritytrading.philadelphia.client.command;
+
+import com.paritytrading.philadelphia.client.TerminalClient;
+import com.paritytrading.philadelphia.client.message.Message;
+import com.paritytrading.philadelphia.client.message.Messages;
+import org.eclipse.collections.api.list.ImmutableList;
+
+import java.util.Scanner;
+
+class WaitCommand implements Command {
+    private static final long WAIT_TIME_MILLIS = 500;
+
+    @Override
+    public void execute(TerminalClient client, Scanner arguments) throws CommandException {
+        if (!arguments.hasNext())
+            throw new CommandException();
+
+        String msgType = arguments.next();
+
+        if (arguments.hasNext())
+            throw new CommandException();
+
+        while(true) {
+            try {
+                Thread.sleep(WAIT_TIME_MILLIS);
+
+                if(lastMsgType(client.getMessages()).equals(msgType))
+                    break;
+
+            } catch(InterruptedException e) {
+                break;
+            }
+        }
```

Comment:
Perhaps we could maintain the existing spacing between a control word (`while`, `if`, `catch`) and opening parenthesis. 🤓

Reply:
Yeah I have no affinity to that syntax so I never see that. Adding 👍